src/optimizer/statistics/expression/propagate_cast.cpp

```diff
@@ -24,14 +24,85 @@ static unique_ptr<BaseStatistics> StatisticsOperationsNumericNumericCast(const B
 static unique_ptr<BaseStatistics> StatisticsNumericCastSwitch(const BaseStatistics &input, const LogicalType &target) {
 	//	Downcasting timestamps to times is not a truncation operation
 	switch (target.id()) {
-	case LogicalTypeId::TIME:
+	case LogicalTypeId::TIME: {
```

Comment:
Brackets?

Reply:
Yes, it was broken before, there was no `break` at the end of the case, so it would fall through
Using brackets makes this a lot more clear imo